tests/kuttl/tests/ovn_restart_flow/03-assert.yaml

```diff
@@ -0,0 +1,27 @@
+#
+# Check for:
+#
+# - The current bridges flows are present in the logs; these flows writen in
+#   the logs are the ones restored from the previous "ovs-vswitchd" container
+#   execution.
+
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 300
+commands:
+    - script: |
+        controller_pod=$(oc get pod -n $NAMESPACE -l service=ovn-controller-ovs -o name|head -1)
+        logs=$(oc logs -n $NAMESPACE -f $controller_pod -c ovs-vswitchd --follow=false)
+        echo "Logs from the controller pod and ovs-vswitchd container:"
+        echo "$logs"
+        flows=$(oc exec -n $NAMESPACE --container ovs-vswitchd $controller_pod -- bash -c 'for br in `ovs-vsctl list-br`; do ovs-ofctl dump-flows $br --no-stats --bundle; done')
+        echo "OVS flows:"
+        echo "$flows"
+        oc exec -n $NAMESPACE --container ovs-vswitchd $controller_pod ovs-vsc
```

Comment:
what is this line doing? Calling `ovs-vsc`? What is it?

Reply:
To be honest, I don't know, I think this is a leftover